README.md

```diff
@@ -45,7 +45,7 @@
   * [dry-rest-permissions](https://github.com/dbkaplan/dry-rest-permissions): Rules based permissions for the Django Rest Framework
 
   ### Documentation
-  * [django-rest-swagger](https://github.com/marcgibbons/django-rest-swagger): Swagger Documentation Generator for Django REST Framework
+  * [django-rest-swagger](https://github.com/marcgibbons/django-rest-swagger): Swagger Documentation Generator for Django REST Framework (deprecated (for Django 2.2+) )
```

Comment:
These few words in deprecation description will result in a false assumption for future folks who will be coming in, If I was one of them I would assume that this package is deprecated for Django 2.2+ but for older versions, it will be ok, which is not right as the package now is completely deprecated regardless of the version and the distinction for Django 2.2+ is only mentioned because the deprecated package won't work out of the box with newer versions of Django because some deprecation changes in Django took place starting from 2.2+.

So, I would prefer to clearly state the package status as deprecated and add the tip about Django 2.2+ issue, so even if someone decided to use it any way he/she knows what he/she is up into.